### ISSUE TITLE
Validation Error in YOLOv8-P6

### ISSUE THREAD

**Issue by yashtiwari96** [ultralytics/ultralytics#6258]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Val

### Bug

The Model is getting trained by using the yolov8-p6.yaml file but an error when using model.val() function. 
![image](https://github.com/ultralytics/ultralytics/assets/72273439/c62aa527-dfef-43d3-a587-aa7ef18b9ab9)

### Minimal Reproducible Example

model=YOLO('yolov8n-p6.yaml')
model.train(data=data, imgsz=752,batch=32)
### Everything is working fine till the above line
validation_model=YOLO('./runs/detect/train/weights/best.pt')
validation_model.val(data=data,imgsz=32)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@yashtiwari96 thanks my friend, yes I am able to reproduce this problem. I'm adding a TODO tag here to fix this. 

Reproduced with:
```
yolo val model=yolov8n-p6.yaml data=coco8.yaml
```

**Comment by glenn-jocher** [comment]
@yashtiwari96 great news 😃! Your original issue may now be resolved ✅ in PR #6261. 

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
